src/integrationTest/resources/tests/arm/properties/all_properties/media-record.properties

```diff
@@ -0,0 +1,40 @@
+#OBJECT_TYPE
+#CASE_NUMBERS
+#FILE_TYPE
+#HEARING_DATE
+#CHECKSUM
+#TRANSCRIPT_REQUEST
+#TRANSCRIPT_TYPE
+#TRANSCRIPT_URGENCY
+#COMMENTS
+#PLACEHOLDER_DATE
```

Comment:
I don't really understand what `PLACEHOLDER_DATE` here signifies. I can't see it used in the Java code mapper stuff?

Reply:
If you check the lld there is a placeholder date. I will remove it as its not required.

Reply:
I honestly couldn't see it on the LLD when I looked earlier (maybe removed) - so in my dizziness left my comment here :)

Reply:
Everything seems to be a constantly sliding doors